assembl/static2/css/components/ideas.scss

```diff
@@ -0,0 +1,104 @@
+.ideas-level {
+    margin: 0 auto;
+    &.animating-towards-inline, &.is-inline {
+        position: relative;
+
+        // scroll containers are an empty area around the scroll button, so that user does not mis-click on idea link which is next to this button
+        .scroll-left-container, .scroll-right-container {
+            display: inline-block;
+            height: 80px;
+            width: 80px;
+            position: absolute;
+            top: calc(50% - 40px);
+            background-color: rgba(0,0,0,0);
+        }
+        .scroll-left-container {
+            left: 0;
+        }
+        .scroll-right-container {
+            right: 0;
+        }
+
+        .scroll-left, .scroll-right {
+            display: none;
+            position: absolute;
+            width: 50px;
+            height: 50px;
+            background-color: rgba(195, 195, 195, 0.6);
+            top: calc(50% - 25px);
```

Comment:
@ayyazdaniaryan the scroll button is vertically centered, so this corresponds to setting a height, and positionning it at 50% of parent height minus half of its height. So here 25 is 50/2, where 50 is the width defined line 25.

@paolina92 OK that's a good idea to use variables for width and height as some things are computed from them. But is the `variables.scss` file the best place to declare them? These width/height variables are completely internal to this component, and are not meant to be used elsewhere. I propose for these cases we declare variables in the component's .scss file instead. What do you guys think?